src/main/scala/firrtl/util/BackendCompilationUtilities.scala

```diff
@@ -93,8 +89,22 @@ trait BackendCompilationUtilities {
                     cppHarness: File
                   ): ProcessBuilder = {
     val topModule = dutFile
-    val command = Seq("verilator",
-      "--cc", s"$dutFile.v") ++
+
+    val blackBoxVerilogList = {
+      val list_file = new File(dir, firrtl.transforms.BlackBoxSourceHelper.FileListName)
+      if(list_file.exists()) {
+        Seq("-f", list_file.getAbsolutePath)
+      }
+      else {
+        Seq.empty[String]
+      }
+    }
```

Comment:
@jackkoenig I agree this could be more elegant, but this works and there are things backed up behind this.  The problem is one we have discussed about the bottom of the toolchain needing to pass stuff back up.  Annotations are a candidate mechanism, there are probably others.  In this case firrtl moves the files, because it knows about what's going on and there is stuff inline that it is well qualified to see.  It does not *know* that these files are being placed there for some subset of the available backends, so the backend knows that if such a file exists that it needs to add a command line options to use them.